### PR TITLE
chore: Update VS Solution Template description

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Other... Please describe:
VS Solution Template

## What is the current behavior?
Missing platforms supported 

## What is the new behavior?
Fixing description adding all platforms currently supported.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
   <TemplateData>
 	<Name>Cross-Platform App (Uno Platform)</Name>
-	<Description>Solution template for creating a cross-platform XAML app with the Uno Platform that targets UWP, Android, iOS, and WebAssembly</Description>
+	<Description>Template for creating a multi-platform C# & XAML application targeting WebAssembly, Android, iOS, macOS and Linux</Description>
```
Comment (sasakrsmanovic) [inline src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:4]: Can we add "Solution" in front of "Template" please. I am also using this to practice gitthub ocllavoration - never approved anything before :)

Comment (sasakrsmanovic) [reply src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:4]: Also - if we can use Multi-platform in place of Cross platform that is better.

File: src/SolutionTemplate/UnoLibraryTemplate/CrossTargetedLibrary.vstemplate [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
   <TemplateData>
 	<Name>Cross-Platform Library (Uno Platform)</Name>
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/UnoLibraryTemplate/CrossTargetedLibrary.vstemplate:3]:
```suggestion
	<Name>Multi-Platform Library (Uno Platform)</Name>
```

File: src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
   <TemplateData>
 	<Name>Cross-Platform App (Uno Platform)</Name>
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:3]:
```suggestion
	<Name>Multi-Platform App (Uno Platform)</Name>
```

File: src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
-	<Description>Solution template for creating a cross-platform XAML app with the Uno Platform that targets UWP, Android, iOS, and WebAssembly</Description>
+	<Description>Solution Template for creating a multi-platform C# & XAML application targeting UWP, WebAssembly, Android, iOS, macOS and Linux</Description>
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:4]:
```suggestion
	<Description>Template for creating a multi-platform C# & XAML application targeting UWP, WebAssembly, Android, iOS, macOS and Linux</Description>
```

Comment (ajpinedam) [reply src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:4]: Removing the "Solution" @davidjohnoliver ?

Comment (davidjohnoliver) [reply src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:4]: @sasakrsmanovic can verify if this is correct

Comment (jeromelaban) [reply src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:4]: Linux is not available in the vsix templates for now.

**Review by sasakrsmanovic — APPROVED** [review]

File: src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate (hunk 4 above)
Comment (jeromelaban) [reply src/SolutionTemplate/UnoSolutionTemplate/UnoApp.vstemplate:4]: It is now :)

**Review by carldebilly — APPROVED** [review]

**Pull request merged**